client/my-sites/plan-features-2023-grid/plan-comparison-grid.tsx

```diff
@@ -471,24 +504,26 @@ export const PlanComparisonGrid: React.FC< PlanComparisonGridProps > = ( {
 					const features = featureGroup.get2023PricingGridSignupWpcomFeatures();
 					const featureObjects = getPlanFeaturesObject( features );
 
+					const featureGroupClass = `feature-group-title-${ featureGroup.slug }`;
+					const groupClasses = classNames( featureGroupClass, {
+						'is-hidden-mobile': ! visibleFeatureGroups.includes( featureGroup.slug ),
+					} );
```

Comment:
This can be done by passing a variable to the components. (Below code assumes that TitleRow was moved to a separate component.)

```typescript
const Row = styled.div< { isHiddenInMobile: boolean } >`
	display: flex;
	justify-content: space-between;
	align-items: center;
	margin-bottom: -1px;
	@media ( max-width: 880px ) {
		display: ${ ( props ) => ( props.isHiddenInMobile ? 'none' : 'flex' ) };
	}
`;
```
And when rendering
```tsx
<Row isHiddenInMobile={ ! visibleFeatureGroups.includes( featureGroup.slug )} >
     <.../>
</Row>
```

Same can be done with the Title component above.

Reply:
Thanks for the suggestion!
I wanted to avoid having to pass the same variable in 3 locations, but this should work too.